text/0092-logs-vision.md

```diff
@@ -0,0 +1,128 @@
+logs, traces and metrics in one package, symmetrically and uniformly for all 3
+types of telemetry data (see also [Collector vision](https://github.com/open-telemetry/opentelemetry-collector/blob/8310e665ec1babfd56ca5b1cfec91c1f997f4f2c/docs/vision.md)).
+
+The unified Collector will support multiple log protocols including the newly
+designed OpenTelemetry log protocol.
+
+Unified collection is important for the following reasons:
+- One agent (or one collector) to deploy and manage.
+- One place of configuration for target endpoints, authentication tokens, etc.
+- Uniform tagging of all 3 types of telemetry data (enrichment by attributes
+  of resources where the data comes from or by user-defined attributes),
+  enabling correct correlation across Resource dimensions later on the backend.
+
+## Cloud Native
+
+We will have best-in-class support for logs emitted in cloud native environments
+(e.g. Kubernetes, serverless, etc), including legacy applications running
+in such environments. This is inline with our CNCF mission.
+
+## Support Legacy
+
+We will produce guidelines on how legacy applications can emit logs in a
+manner that makes them compatible with OpenTelemetry's approach and enables
+telemetry data correlation. We will also have a reasonable story around
+logs that are emitted by sources over which we may have no control and which
+emit logs in pre-defined formats via pre-defined mediums (e.g. flat file logs,
+Syslog, etc).
+
+We will have technical solutions or guidelines for using popular logging
+libraries in OpenTelemetry-compatible manner and we may produce logging
+libraries for languages where gaps exist.
+
+This is important because we believe software that was created before
+OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
+efforts where possible.
+
+### Auto-instrumentation
+
+To enable functionality that requires modification of how logs are emitted we
+will work on auto-instrumenting solutions. This will reduce the adoption barrier
+for existing deployments.
+
+### Applicable to All Log Sources
+
+Logging support at OpenTelemetry will be applicable to all sorts of log sources:
+system logs, infrastructure logs, third-party and first-party application logs.
+
+### Standalone and Embedded Logs
+
+OpenTelemetry will support both logs embedded inside [Spans](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#span)
+and standalone logs recorded elsewhere. The support of embedded logs is
+important for OpenTelemetry's primary use cases, where errors and exceptions
+need to be embedded in Spans. The support of standalone logs is important for
+legacy applications which may not emit Span's at all.
+
+## Prior art and alternatives
+
+Prior art in logging industry is nearly impossible to enumerate. There are
+countless logging libraries, collection agents, network protocols, open-source
+and proprietary backends.
```

Comment:
You could still list a few of the most important/popular ones.

Reply:
This is poorly worded. Let me rewrite the section.

Reply:
Rewrote the section.